webapp/channels/src/components/admin_console/ldap_wizard/ldap_wizard.tsx

```diff
@@ -547,27 +635,59 @@ const LDAPWizard = (props: Props) => {
 
         return (
             <div>
-                {sections}
+                {renderedSections}
             </div>
         );
     };
 
     return (
         <div
-            className={'wrapper--fixed'}
+            className={'wrapper--fixed ldap-wizard-wrapper'}
             data-testid={`sysconsole_section_${schema?.id}`}
         >
             {renderTitle()}
-            <div className='admin-console__wrapper'>
-                <div className='admin-console__content'>
-                    <form
-                        className='form-horizontal'
-                        role='form'
-                        onSubmit={handleSubmit}
-                    >
-                        {renderSettings()}
-                    </form>
-                    {hybridSchemaAndComponent()}
+            <div className='ldap-wizard-content-wrapper'>
+                <div className='ldap-wizard-sidebar'>
+                    <div className='ldap-wizard-sidebar-header'>
+                        <i className='icon icon-text-box-outline'/>
+                        <FormattedMessage
+                            id='admin.ldap_wizard.sections_header'
+                            defaultMessage='Sections'
+                        />
+                    </div>
+                    {memoizedSections.map((section) => (
+                        <div
+                            key={section.key + '-sidebar-item'}
+                            className={`ldap-wizard-sidebar-item ${section.key === activeSectionKey ? 'ldap-wizard-sidebar-item--active' : ''}`}
+                            onClick={() => {
+                                const sectionElement = sectionRefs.current[section.key];
+                                if (sectionElement) {
+                                    sectionElement.scrollIntoView({behavior: 'smooth', block: 'start'});
+                                }
+                            }}
+                            role='button'
```

Comment:
Could you make this into a button instead of having it be a div that behaves as a button? That'd save us from having to specify most of this since it would get that for free